cpp/log/cert_test.cc

```diff
@@ -85,6 +85,40 @@ static const char kInvalidCertString[] =
 
 static const char kMismatchingSigAlgsCertString[] =
```

Comment:
Please add a comment saying this certificate was derived from kIllegalSigAlgParameterCertString with the unsigned parts changed.